core/sequencer/sequencer_api.proto

```diff
@@ -75,8 +70,29 @@ message DefineRevisionsRequest {
 
 // DefineRevisionResponse contains information about freshly defined revisions.
 message DefineRevisionsResponse {
-  // outsanding_revisions a list of all the defined revisions which are not yet applied.
-  repeated int64 outstanding_revisions = 1;
+  // Reserved for the deprecated outstanding_revisions field.
+  reserved 1;
+  // begin is the first outstanding revision (inclusive).
+  int64 begin = 2;
+  // end is the last outstanding revision (exclusive). It is equal to begin if
+  // there are no outstanding revisions.
+  int64 end = 3;
+}
+
+// GetNewRevisionsRequest contains information needed to query the set of
+// outstanding revisions.
+message GetNewRevisionsRequest {
```

Comment:
Maybe "GetUnappliedRevisionsRequest/Response" or "GetOutstandingRevisionsRequest/Response"? "New" could be a misnomer, as revisions may be defined but not applied for a long time. Whatever you pick, be consistent with the field naming above.

Reply:
Done.